components/Service/ServicePage.js

```diff
@@ -0,0 +1,172 @@
+                                <p className="intro-text1">
+                                Medicial Screenings
+                                </p>
+                                <p className="intro-text2">{INTRO_TEXT2}</p>
+                            </span>
+                        </div>
+                        <div className="intro-content">
+                            <i className="fas fa-ambulance icons-status" />
+                            <hr />
+                            <span className="services-content-text context1">
+                                <p className="intro-text1">
+                                   Emergency Ambulance
+                                </p>
+                                <p className="intro-text2">{INTRO_TEXT2}</p>
+                            </span>
+                        </div>
+                    </div>
+                </div>
+            </div>
+            <div className="services app-content">
+                <div className="content-wrapper services-wrapper service-wrap">
+                    <div className="services-content">
+                        <img className="serviceI" src="/static/img/help.jpg" alt="Smiley face" />
+                        <span className="services-content-text">
+                            <p className="service-text1">
+                                Emergency Contact
+                                <br />
+                                {' '}
+                                Number
+                            </p>
+                            <p className="service-text2">{HELP_DESK}</p>
+                        </span>
+                    </div>
+                    <div className="services-content">
+                        <img className="serviceI" src="/static/img/cardiac.jpg" alt="Smiley face" />
+                        <span className="services-content-text">
+                            <p className="service-text1">
+                                Instructions On How To
+                                <br />
+                                {' '}
+                                Administer First Aid
+                            </p>
+                            <p className="service-text2">{HELP_DESK}</p>
+                        </span>
+                    </div>
+                    <div className="services-content">
+                        <img className="serviceI" src="/static/img/care.jpg" alt="Smiley face" />
+                        <span className="services-content-text">
+                            <p className="service-text1">
+                                Opportunity To Ask Doctors
+                                <br />
+                                {' '}
+                                Questions
+                            </p>
+                            <p className="service-text2">{HELP_DESK}</p>
+                        </span>
+                    </div>
+                </div>
+            </div>
```

Comment:
map over these contents

Reply:
Will do that once I start connecting it to the api directly